src/includes/getting-started-primer/javascript.nextjs.mdx

```diff
@@ -0,0 +1,14 @@
+<Note>
+
+_Sentry's Next.js SDK enables automatic reporting of errors and exceptions._
+
+</Note>
+
+Features:
+
+- Automatic [Error Tracking](/product/issues/) with Source Maps for both JavaScript and TypeScript
+- Events [enriched](/platforms/javascript/enriching-events/context/) with device data
+- [Breadcrumbs](/platforms/javascript/enriching-events/breadcrumbs/) created for outgoing `http` request with XHR and Fetch; UI and system events; and console logs
+- [Release Health](/product/releases/health/) tracks crash free users and sessions
+- Under the hood the SDK relies on our [React SDK](/platforms/javascript/guides/react/) on the frontend and [Node SDK](/platforms/node) on the backend, which makes all functions available for React and Node also available in this SDK
+- On Device symbolication for JavaScript (in Debug)
```

Comment:
Should this read `On-device`?  cc @PeloWriter

@iker-barriocanal what does "in Debug" mean here?

Reply:
This shouldn't be here, removing it.